src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/ProgrammableAndroid.java

```diff
@@ -470,7 +470,12 @@ public void openScriptEditor(Player p, Block b) {
 
         menu.addItem(1, new CustomItem(HeadTexture.SCRIPT_FORWARD.getAsItemStack(), "&2> Edit Script", "", "&aEdits your current Script"));
         menu.addMenuClickHandler(1, (pl, slot, item, action) -> {
-            openScript(pl, b, getScript(b.getLocation()));
+            if (PatternUtils.DASH.split(BlockStorage.getLocationInfo(b.getLocation()).getString("script")).length <= MAX_SCRIPT_LENGTH) {
+                openScript(pl, b, getScript(b.getLocation()));
+            } else {
```

Comment:
This check is already performed in #setScript(...).
So no real need for that.

Reply:
That's the entire reason I made the pr. I have the need for scripts of 100 instructions or more. setscript won't allow me to do that but I can use blockstorage to circumvent that. However if a player edits a script of such a big size they will get an inventory error. That's why I added the second check

Reply:
:NotLikeThis:

Reply:
So...